File: vendaval.client/src/app/chat/chatuser.ts
import { UserType } from "../shared/common/enums/user-type";
import { BaseModel } from "../shared/common/interfaces/BaseModel";

export interface ChatUser extends BaseModel {
  connectionId: string;
  name: string;
  email: string;
  userType: UserType;
  isOnline: boolean;
}

File: Vendaval.Domain/Entities/ChatUser.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Vendaval.Domain.Enums;

namespace Vendaval.Domain.Entities
{
    public class ChatUser : BaseModel
    {
        public string ConnectionId { get; set; }
        public UserType UserType { get; set; }
        public string Email { get; set; }
        public string Name { get; set; }
        public bool IsOnline { get; set; }
    }
}
